cookbooks/aws-parallelcluster-config/templates/default/init/fetch_and_run.erb

```diff
@@ -84,6 +103,8 @@ function run_postupdate () {
 check_params
 
 ACTION=${1#?}
+custom_script=""
+url_len_limit=100
```

Comment:
Could you add a comment in the code to exmplain the 100 limit here?

Reply:
comment added